### DISCUSSION TITLE
Internationalisation/Localisation

### DISCUSSION THREAD

**Discussion by kcramakrishna** [Digital-Assistant/Digital_Assistant_Client#283] [Ideas]
There seems to be numerous libraries:
https://openbase.com/categories/js/best-nodejs-localization-libraries
https://phrase.com/blog/posts/node-js-i18n-guide/

To be researched and finalised.

**Comment by kcramakrishna** [comment]
We decided to use i18n for this purpose.

**Reply by lakshmanveti** [reply]
Since we are using React, React has its own supportive framework "react-i18next" which is based on "i18next".
[https://react.i18next.com/](url)

**Comment by yureshwar** [comment]
For nestjs this is the library: https://nestjs-i18n.com/ 

i18next can be used for nodejs also: https://www.i18next.com/

Comparision of both is here https://g.co/gemini/share/0988d41801f2

Summary 

![Screenshot from 2024-07-26 11-12-24](https://github.com/user-attachments/assets/21bee98f-8ee9-4bea-9881-d089892b8222)
